spec/mock-request.js

```diff
@@ -59,21 +75,24 @@ HttpBackend.prototype = {
                     console.log("  [%s] Flushed assigned amount: %s", path, numToFlush);
                     defer.resolve();
                 } else {
-                    setTimeout(tryFlush, 0);
+                    // setImmediate(tryFlush);
+                    process.nextTick(tryFlush);
+                    // tick(0);
```

Comment:
Presumably you're doing this to get around the fact that we're now mocking time so `setTimeout` isn't working as we expect. Can you add a comment to this effect please.